server/views/pages/summary.njk

```diff
@@ -102,6 +102,59 @@
         </div>
     </section>
 
+    <section class="app-summary-card govuk-!-margin-bottom-6 app-summary-card--large-title" id ="case-details">
+        <header class="app-summary-card__header">
+            <h2 class="app-summary-card__title">Case details</h2>
+        </header>
+        <div class="app-summary-card__body">
+            {{ govukSummaryList({
+                rows: [
+                    {
+                        key: {
+                        text: "PNC"
+                    },
+                        value: {
+                        text: data.pncNumber
+                    }
+                    },
+                    {
+                        key: {
+                        text: "CPS pack"
+                    },
+                        value: {
+                        html: '<span class="moj-badge moj-badge--red moj-badge--large">Missing</span>'
+                    }
+                    },
+                    {
+                        key: {
+                        text: "Pre-convictions"
+                    },
+                        value: {
+                        html: '<span class="moj-badge moj-badge--red moj-badge--large">Missing</span>'
+                    }
+                    },
+                    {
+                        key: {
+                        html: 'Pre-sentence report' + '<br><span class="govuk-!-font-size-16 govuk-!-font-weight-regular">' + data.courtReport.description + '</span>'
+                    },
+                        value: {
+                        text: data.courtReport.completedDate | dateFormat
+                    }
+                    },
+                    {
+                        key: {
+                        text: 'Last OASys assessment'
+                    },
+                        value: {
+                        text: data.assessment.lastAssessedOn | dateFormat
```

Comment:
I'm unsure if this scenario is only in lower environments but there is a chance the assessment might not exist too

Reply:
Ok, no probs, I'll do that now 👍